support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -417,9 +417,12 @@ AccessControlled accessControlled() {
                         cookie = UUID.randomUUID().toString();
                         // Switches the label to a self-label, so if the executable is killed and restarted via ExecutorPickle, it will run on the same node:
                         label = computer.getName();
+
```

Comment:
IMO these are fine, as it improves readability by making the environment changes (which are extended by this PR) into their own paragraph.